src/main/java/com/gregtechceu/gtceu/api/machine/fancyconfigurator/MEFancyInvConfigurator.java

```diff
@@ -0,0 +1,68 @@
+package com.gregtechceu.gtceu.api.machine.fancyconfigurator;
+
+import com.gregtechceu.gtceu.api.gui.GuiTextures;
+import com.gregtechceu.gtceu.api.gui.fancy.IFancyConfigurator;
+
+import com.lowdragmc.lowdraglib.gui.texture.IGuiTexture;
+import com.lowdragmc.lowdraglib.gui.widget.SlotWidget;
+import com.lowdragmc.lowdraglib.gui.widget.Widget;
+import com.lowdragmc.lowdraglib.gui.widget.WidgetGroup;
+import com.lowdragmc.lowdraglib.jei.IngredientIO;
+import com.lowdragmc.lowdraglib.misc.ItemStackTransfer;
+
+import net.minecraft.network.chat.Component;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.experimental.Accessors;
+
+import java.util.Collections;
+import java.util.List;
+
+@Accessors(chain = true)
+public class MEFancyInvConfigurator implements IFancyConfigurator {
```

Comment:
just name this `FancyInvConfigurator`, since it isn't specific to ME parts in any way.